src/scenes/Scene.js

```diff
@@ -2,13 +2,13 @@
  * @author mrdoob / http://mrdoob.com/
  */
 
-THREE.Scene = function () {
+THREE.Scene = function ( fog ) {
```

Comment:
@mrdoob What is the decision on adding `fog` as an argument to the `Scene` constructor?